src/Components/SearchBar.js

```diff
@@ -71,25 +71,34 @@ if (Privacy.isAllowed(PRIVACY_ACTIONS.SEARCH)) {
                             .collections(this.props.index)
                             .documents()
                             .search(options)
-                            .then(res => {
+                            .then((res) => {
                                 callback(res.hits);
                             })
-                            .catch(e => {
+                            .catch((e) => {
                                 rethrow(e);
                             });
                     },
                     templates: {
                         suggestion:
                             this.props.suggestion_template ||
-                            function(suggestion) {
+                            function (suggestion) {
                                 let d = suggestion.document;
 
-                                let name_hs = suggestion.highlights.filter(a => a.field === 'name');
-                                let runs_hs = suggestion.highlights.filter(a => a.field === 'runs');
+                                let name_hs = suggestion.highlights.filter((a) => a.field === 'name');
+                                let runs_hs = suggestion.highlights.filter((a) => a.field === 'runs');
 
                                 return (
                                     '<span><strong>' +
                                     (name_hs.length === 1 ? name_hs[0].snippet : d.name) +
+                                    (d.quality === 'tested'
```

Comment:
FF a11y checking tool doesn't like this: 
![grafik](https://user-images.githubusercontent.com/45886910/98440494-53cfb980-20f9-11eb-8543-5d1a17b0f094.png)